src/pip/_internal/commands/search.py

```diff
@@ -14,7 +14,7 @@
 from pip._internal.cli.base_command import Command
 from pip._internal.cli.req_command import SessionCommandMixin
 from pip._internal.cli.status_codes import NO_MATCHES_FOUND, SUCCESS
-from pip._internal.download import PipXmlrpcTransport
+from pip._internal.network.xmlrpc import PipXmlrpcTransport
```

Comment:
We'd want to move this to be alphabetically sorted. That's why the linter is failing.

Reply:
oh. I forgot to run `isort` in this file. Sure, let me update